frontend/views/pages/Contributions.vue

```diff
@@ -2,164 +2,146 @@
 page(pageTestName='contributionsPage' pageTestHeaderName='contributionsTitle')
   template(#title='') {{ L('Contributions') }}
 
-  // v-if='memberProfile(ourUsername).groupProfile.incomeDetailsKey'
   page-section.c-card-empty
     svg-contributions.c-svg
     div
       i18n(tag='h3') Add your income details
       i18n.c-text(tag='p') This will allow you to start receiving or giving mincome.
       i18n(tag='button' @click='openModal("IncomeDetails")') Add income details
 
-  section.card
-    i18n(tag='h2' class='card-header') Receiving
-
-    ul.c-ul
-      contribution(
-        v-for='(contribution, index) in fakeStore.receiving.nonMonetary'
-        :key='`contribution-${index}`'
-      )
-        //- TODO: verify this is safe (no XSS)
-        span(v-html='textReceivingNonMonetary(contribution)')
-        text-who(:who='contribution.who')
-
-      contribution(
-        v-if='doesReceiveMonetary'
-        variant='editable'
-        ismonetary=''
-        @interaction='handleFormTriggerClick'
-      )
-        //- TODO: verify this is safe (no XSS)
-        span(v-html='textReceivingMonetary(fakeStore.receiving.monetary)')
-        text-who(:who='fakeStore.groupMembersPledging')
-        i18n each month
-
-  section.card
-    i18n(tag='h2' class='card-header') Giving
-    ul
-      contribution.has-text-weight-bold(
-        v-for='(contribution, index) in fakeStore.giving.nonMonetary'
-        :key='`contribution-${index}`'
-        variant='editable'
-        @new-value='(value) => handleEditNonMonetary(value, index)'
-      ) {{contribution}}
-
-      contribution(
-        variant='unfilled'
-        @new-value='submitAddNonMonetary'
+  .c-contribution-header
+    .has-text-1
+      i18n(
+        v-if='fakeStore.receiving.monetary'
+        tag='p'
+        :args='{upTo: fakeStore.upTo, currency: fakeStore.currency}'
+        html='You need<span class="has-text-bold has-text-0">{currency}{upTo}</span>'
```

Comment:
If `html` is used `i18n` tags will give a warning since there is a new way of using HTML with them that should be used, see the example here: #727